Automattic/jetpack
# PHPCS: SSO

#### Changes proposed in this Pull Request:

PHPCS cleanup for SSO module.

#### Jetpack product discussion

Closes #21971

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Proofread changes. I've tried breaking down the types of changes into individual commits for easier reading.
* Enable SSO on a test site, play around with the module & make sure nothing breaks :)
